src/Migrations/Migrations.csproj

```diff
@@ -116,6 +107,10 @@
       <Private>False</Private>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <Folder Include="DynamoMSOffice\" />
```

Comment:
Those files were automatically by Visual Studio added after I deleted the .cs files in them. 
I have removed them and updated the PR.